### PR TITLE
Release 3.1.1

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+# 3.1.1 (2019-06-10)
+
+- Fixed the flyout component to handle click events that originate within its contents. [#44](https://github.com/blackbaud/skyux-flyout/pull/44)
+- Fixed the flyout component to style the resize handle and iterator buttons correctly in Internet Explorer. [#45](https://github.com/blackbaud/skyux-flyout/pull/45)
+- Fixed the flyout component to only resize to twenty pixels less than the window width. [#46](https://github.com/blackbaud/skyux-flyout/pull/46)
```
Comment (Blackbaud-AlexKingman) [inline CHANGELOG.md]: How about something like "Fixed the flyout component to never resize larger than twenty pixels less than the window?" Docs, feel free to chime in here...

Comment (blackbaud-johnly) [reply CHANGELOG.md]: How about "Fixed the flyout component to limit its maximum width to 20 pixels less than the window's width."

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+- Fixed the flyout component to use a default width of half the window size when no default width is specified by consumers. [#47](https://github.com/blackbaud/skyux-flyout/pull/47)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: It's just minor tweaks, but I'd change this to "Fixed the flyout component to set the default width to half of the window's width unless consumers specify a different default."

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,12 @@
...
+- Fixed the flyout component to properly handle when a default width which is less than the minimum width is given. [#49](https://github.com/blackbaud/skyux-flyout/pull/49)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: Again, it's just minor tweaks, but I'd change this to "Fixed the flyout component to handle a default width that is less than the minimum width."

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: That's fine

File: CHANGELOG.md (hunk 2 above)
Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: That's fine

File: CHANGELOG.md (hunk 3 above)
Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: That's fine

**Comment by Blackbaud-TrevorBurch** [comment]
@blackbaud-johnly I'm fine if you want to make the suggested changes

**Pull request merged**